src/main/g8/src/main/scala/com/lightbend/akka/http/sample/QuickstartServer.scala

```diff
@@ -0,0 +1,45 @@
+package com.lightbend.akka.http.sample
+
+import akka.actor.{ ActorRef, ActorSystem }
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.Http.ServerBinding
+import akka.http.scaladsl.server.Route
+import akka.stream.ActorMaterializer
+import akka.util.Timeout
+
+import scala.concurrent.{ ExecutionContext, Future }
+import scala.concurrent.duration._
+import scala.io.StdIn
+
+//#main-class
+object QuickstartServer extends App
+    with UserRoutes {
+
+  // set up ActorSystem and other dependencies here
+  //#main-class
+  //#server-bootstrapping
+  implicit val system: ActorSystem = ActorSystem("helloAkkaHttpServer")
+  implicit val materializer: ActorMaterializer = ActorMaterializer()
+  //#server-bootstrapping
+
+  // Needed for the Future and its methods flatMap/onComplete in the end
+  implicit val executionContext: ExecutionContext = system.dispatcher
+
+  val userRegistryActor: ActorRef = system.actorOf(UserRegistryActor.props, "userRegistryActor")
+
+  //#main-class
+  lazy val routes: Route =
+    userRoutes // from the UserRoutes trait
+  //#main-class
+
+  //#http-server
+  val serverBindingFuture: Future[ServerBinding] = Http().bindAndHandle(routes, "localhost", 8080)
+  println(s"Server online at http://localhost:8080/\nPress RETURN to stop...")
+  StdIn.readLine()
+  serverBindingFuture
+    .flatMap(_.unbind())
+    .onComplete(_ => system.terminate())
```

Comment:
Eating errors here means that a beginner will not get any info that the binding failed because there already was a running something on that port. Log the error case perhaps?